### PR TITLE
feat(#332): ability to enable hotwire

### PR DESCRIPTION
**Pull request by przbadu**
## Issue
#332 

## What happened

It prompt us to enable hotwire in newly generating rails application, if the answer is `Y` then it will add
- turbo-rails
- stimulus-rails

and enable them in the rails project.

## Proof Of Work

image

image

### CODE REVIEW

File: .template/addons/hotwire/Gemfile.rb [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Hotwire Rails
+insert_into_file 'Gemfile', after: /gem 'bcrypt'.*\n/ do
+  <<~RUBY.indent(2)
```
Comment (malparty) [inline .template/addons/hotwire/Gemfile.rb:3]:
```suggestion
  <<~RUBY
```

Hi @przbadu  and Thanks for your contribution! :) 🙌 

I've tested it locally, the prompt is nice :) What about removing the indent as these lines are not in a group? 

For illustration, here is the resulting indentation I had at the moment:
image


Another suggestion open for your ideas, should we add a small comment before these two lines (something like `# Hotwire`) — asking @rosle @bterone  as well to share opinions :) 

Thanks!

Comment (przbadu) [reply .template/addons/hotwire/Gemfile.rb]: @malparty 

Thank you, yes those changes make sense to me.

**Review by malparty — APPROVED** [review]
Looks good to me! :)

**Review by bterone — APPROVED** [review]
Thanks for your contribution 🙌

**Review by hanam1ni — APPROVED** [review]

**Review by olivierobert — APPROVED** [review]

**Pull request merged**